bin/install_nvidia.sh

```diff
@@ -302,7 +302,8 @@ pip3 install git+https://github.com/mapillary/inplace_abn.git
 pip3 install git+https://github.com/facebookresearch/detectron2.git@4841e70ee48da72c32304f9ebf98138c2a70048d
 pip3 install git+https://github.com/cidl-auth/DCNv2
 pip3 install ${OPENDR_HOME}/src/opendr/perception/panoptic_segmentation/efficient_ps/algorithm/EfficientPS
-pip3 install ${OPENDR_HOME}/src/opendr/perception/panoptic_segmentation/efficient_ps/algorithm/EfficientPS/efficientNet
+pip3 install ${OPENDR_HOME}/src/opendr/perception/panoptic_segmentation/efficient_lps/algorithm/EfficientLPS
```

Comment:
I don't believe this script for embedded platforms relies on `parse_dependencies.py` as the dependencies are all hard-coded, so I suspect the issue there would persist. So I think the other solution would need to be implemented in this case

Reply:
I have switched this to EfficientLPS because the reason I mentioned as EfficientLPS can in fact not be able to use geffnet installation from EfficientPS submodule and be on the safe side. Unfortunately I am not an expert of how this script is used. I had a look at this because this was mentioned in the issue 😅.

What would be your suggestion here ?

Reply:
Sorry I was under the impression that both `efficientNet` were being installed again but I must have read it wrong, my bad!